diff --git a/node_modules/@types/decompress/index.d.ts b/node_modules/@types/decompress/index.d.ts
index 408e512..d513570 100755
--- a/node_modules/@types/decompress/index.d.ts
+++ b/node_modules/@types/decompress/index.d.ts
@@ -4,8 +4,6 @@
 //                 Jesse Bethke <https://github.com/jbethke>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-/// <reference types="node" />
-
 export = decompress;
 
 declare function decompress(input: string | Buffer, output?: string | decompress.DecompressOptions, opts?: decompress.DecompressOptions): Promise<decompress.File[]>;
